recipes/freecadnoui/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "freecadnoui" %}
+{% set version = "1.0" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/Lordling/FreeCADNoUI/archive/freecadnoui.tar.gz
```

Comment:
I just saw that you don't have a release, please generate one to use here

Reply:
Well.. I just saw that you add a ``.tar.gz`` file in your git project. Would be good to just let your code there and generate tag release to the version which you want to